src/Provider/Pelias/Pelias.php

```diff
@@ -132,48 +133,62 @@ protected function executeQuery(string $url): AddressCollection
 
         $results = [];
         foreach ($locations as $location) {
-            if (isset($location['bbox'])) {
-                $bounds = [
-                    'south' => $location['bbox'][3],
-                    'west' => $location['bbox'][2],
-                    'north' => $location['bbox'][1],
-                    'east' => $location['bbox'][0],
-                ];
-            } else {
-                $bounds = [
-                    'south' => null,
-                    'west' => null,
-                    'north' => null,
-                    'east' => null,
-                ];
-            }
+            $results[] = $this->buildAddress($location);
+        }
 
-            $props = $location['properties'];
+        return new AddressCollection($results);
+    }
 
-            $adminLevels = [];
-            foreach (['region', 'county', 'locality', 'macroregion', 'country'] as $i => $component) {
-                if (isset($props[$component])) {
-                    $adminLevels[] = ['name' => $props[$component], 'level' => $i + 1];
-                }
-            }
+    /**
+     * Build the Address object from the the Feature
+     * @param array $location the Feature array
+     * @return Address the address object
+     */
+    protected function buildAddress(array $location)
+    {
+        if (isset($location['bbox'])) {
+            $bounds = [
+                'south' => $location['bbox'][3],
+                'west' => $location['bbox'][2],
+                'north' => $location['bbox'][1],
+                'east' => $location['bbox'][0],
+            ];
+        } else {
+            $bounds = [
+                'south' => null,
+                'west' => null,
+                'north' => null,
+                'east' => null,
+            ];
+        }
```

Comment:
```suggestion
        $bounds = [
            'south' => $location['bbox'][3] ?? null,
            'west' => $location['bbox'][2] ?? null,
            'north' => $location['bbox'][1] ?? null,
            'east' => $location['bbox'][0] ?? null,
        ];
```

Reply:
My goal was to change the original Provider as minimally as possible.
However this is a nice touch to make the code more readable, so I have included your suggestion.